contracts/bond/Bond.sol

```diff
@@ -90,9 +95,10 @@ contract Bond is
     );
 
     /**
```

Comment:
I guess /** or /* depends on if we have sections like "notice" or "param" or not?